clam/clamservice.py

```diff
@@ -1404,6 +1418,8 @@ def start(project, credentials=None): #pylint: disable=too-many-return-statement
                     cmd = "ssh -o NumberOfPasswordPrompts=0 " + settings.REMOTEUSER + "@" + settings.REMOTEHOST + " " + cmd
                 else:
                     cmd = "ssh -o NumberOfPasswordPrompts=0 " + settings.REMOTEHOST + " " + cmd
+
+            Project.updateindex(user, project, 0.0, clam.common.status.RUNNING)
```

Comment:
Hmm, I wonder why this wasn't here already in the first place, that's indeed a bug and means that the running status hasn't been propagated to the index list for quite a while, I hadn't noticed. I'm moving this a few lines down though so it only runs when the dispatcher has been succesfully launched (running asynchronously).